src/content/docs/integrations/prometheus-integrations/install-configure/remote-write-drop-data.mdx

```diff
@@ -0,0 +1,96 @@
+
+If a target is sending a noisy metric that you don't want sent to New Relic, you can specify that New Relic should drop that data. 
+
+### Example 
+
+Let's say you don't want to recieve data for the metric `node_memory_active_bytes` from an instance running at `localhost:9100`. Using the `write_relabel_config` entry shown below, you can target the metric name using the `__name__` label in combination with the instance name.
+
+```
+remote_write:
+- url: https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=macbook-server-cluster
+  bearer_token: <redacted>
+  write_relabel_configs:
+    - source_labels: ['__name__', 'instance']
+      regex: 'node_memory_active_bytes;localhost:9100'
+      action: 'drop'
+```
+
+This tells Prometheus that you want to do some action against metrics with these labels. To limit which metrics with these labels are affected you must include some value for regex. By default this value is set to `.*` so will include all metrics. In this case, it will drop all metric data points coming out of Prometheus via remote write. 
+
+## Drop specific labels or attributes from data points  [#drop-specific]
+
+If a target is sending specific labesl or attributes you're not interested in receiving, you can drop these from the metrics you receive. 
+
+### Example
+
+Let's say one of your targets is sending a bunch of extra attributes you're not interested in receiving. These might include things like high cardinality attributes such as unique machine identifiers, JVM IDs, or similar. In this case, you need to change both the `remote_write` and the `scrape_configs` section of the YAML file. The result will look something like this. 
+
+```
+remote_write:
+- url: https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=macbook-server-cluster
+  bearer_token: <redacted>
+  write_relabel_configs:
+    - regex: 'extraLabelToRemove.*'
+      action: 'labeldrop'
+...
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: 'node'
+    # Override the global default and scrape targets from this job every 5 seconds.
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['localhost:9100']
+        labels:
+          group: 'production'
+          keepLabelName1: 'please-keep-me'
+          extraLabelToRemove: 'please-remove-me'
+          extraLabelToRemove1: 'please-remove-me'
+          extraLabelToRemove2: 'please-remove-me'
+          extraLabelToRemove4: 'please-remove-me'
+          extraLabelToRemove3: 'please-remove-me'
+          extraLabelToRemove5: 'please-remove-me'
+```
+
+## Prometheus or NerdGraph? [#prometheus-vs-nerdgraph]
+
+There are advantages to both dropping data using the method described on this page and using using [NerdGraph](/docs/accounts/accounts/data-management/drop-data-using-nerdgraph/). This section is intended to help you figure out which method is better for your specific needs and preferences. 
```

Comment:
Line 73: You repeat `using` twice. Remove one of them

Reply:
Fixed!